### PR TITLE
Add rc

### PR DESCRIPTION
**Pull request by ralphtheninja**
Ticks off first part of https://github.com/deltachat/deltachat-desktop/issues/550

### CODE REVIEW

**Comment by ralphtheninja** [comment]
@Simon-Laux Once this has been merged, we can add default configuration for the logger to `src/rc.js`.

File: src/main/index.js [hunk 1]
```diff
@@ -26,18 +27,12 @@ process.on('exit', function () {
 
 // Ensure CONFIG_PATH exists.
 mkdirp.sync(config.CONFIG_PATH)
-/* *CONFIG* */
-let argv = sliceArgv(process.argv)
 
 if (config.IS_PRODUCTION) {
   // When Electron is running in production mode (packaged app), then run React
   // in production mode too.
   process.env.NODE_ENV = 'production'
```
Comment (ralphtheninja) [inline src/main/index.js]: @Jikstra I wonder if this is needed or even matters. Do you know if `NODE_ENV` means anything to react when in production?

Comment (ralphtheninja) [reply src/main/index.js]: The more we can get rid of small complexities/oddities like this, the better. This code doesn't say anything about _why_ this is important.

File: src/rc.js [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = require('rc')('DeltaChat', {})
```
Comment (ralphtheninja) [inline src/rc.js]: @Simon-Laux Just add some properties here with sane defaults for the logger.

File: src/main/index.js (hunk 1 above)
Comment (Jikstra) [reply src/main/index.js]: > @Jikstra I wonder if this is needed or even matters. Do you know if `NODE_ENV` means anything to react when in production?

Would be suprised about that, as far as i know the process global isn't defined in browser js. Let's remove and see, can add it back if we need to. We aren't using NODE_ENV anyway so who cares.

Comment (ralphtheninja) [reply src/main/index.js]: What I'm actually getting at is if we can remove `config.IS_PRODUCTION` :)

File: src/rc.js (hunk 2 above)
Comment (Simon-Laux) [reply src/rc.js]: Ok, I will do that If you didn't already let's see

Comment (Simon-Laux) [reply src/rc.js]: Update: looks like we have to first fix the issue I mentioned here: https://github.com/deltachat/deltachat-desktop/pull/577#issuecomment-453229778

**Pull request merged**